### PR TITLE
🌱 Bump CAPI to v1.3.3

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

File: test/go.mod [hunk 1]
```diff
@@ -14,12 +14,12 @@ require (
...
 	k8s.io/client-go v0.25.4
 	k8s.io/utils v0.0.0-20220823124924-e9cbc92d1a73
-	sigs.k8s.io/cluster-api v1.3.2
+	sigs.k8s.io/cluster-api v1.3.3
 	sigs.k8s.io/cluster-api/test v1.3.2
 	sigs.k8s.io/controller-runtime v0.13.1
 )
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.3.2
+replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v1.3.3
```
Comment (furkatgofurov7) [inline test/go.mod:22]: can you pls update the test module as well on line 24?

**Review by furkatgofurov7 — APPROVED** [review]
/test-ubuntu-integration-main

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Pull request merged**